pymesos/process.py

```diff
@@ -143,14 +147,16 @@ def abort(self):
 
     def stop(self):
         Process.abort(self)
-        self.join()
-        for addr in self.conn_pool:
-            self.conn_pool[addr].close()
-        self.conn_pool.clear()
+        if threading.current_thread() != self.accept_t:
```

Comment:
`accept_t` is  a internal thread, so I think it's impossible to call `stop` in this thread out of pymesos. According to official documents, we must allow to call `driver.stop()` in scheduler's api which runs in `delay_t` thread..